### PR TITLE
Add some stuff to README.MD

### PR DESCRIPTION
**Pull request by PassiHD2004**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,24 @@ TODO: add a list of features
 ### Can I use this on a Vanilla server?
 Yeah sure. FoxClient is 100% client side and doesn't require anything to be installed on the server.
 
+### How can i use the latest Features?
```
Comment (RoootTheFox) [inline README.md:11]: This section is empty

File: README.md [hunk 2]
```diff
@@ -8,6 +8,24 @@ TODO: add a list of features
...
+
+#### Please note that the very latest Version may be unstable!
```
Comment (RoootTheFox) [inline README.md:13]: keeping words lowercase might be a good idea

**Review by RoootTheFox — CHANGES_REQUESTED** [review]
Needs some changes, looks good though

File: README.md (hunk 1 above)
Comment (RoootTheFox) [reply README.md:11]: Actually, I'd just change the name of this section to "Build instructions" or something

**Comment by PassiHD2004** [comment]
fixed

**Review by RoootTheFox — APPROVED** [review]
Looks good, lgtm

**Pull request merged**